src/schema/v1/artwork/index.ts

```diff
@@ -211,6 +211,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      editionSetById: {
```

Comment:
We're still stitching into the v1 schema, so we need this here as well.

Reply:
Currently this takes an `id` argument, but might expand to include other(?) arguments. I would just name it `editionSet` (not sure if that conflicts already?), rather than encode `...ByID` in the field name.

Reply:
I actually liked how explicit this was, but I see your point! Mine is not a strong opinion and you're right that we don't have a lot of examples of this `byId` pattern. Will update.